k8s_handle/__init__.py

```diff
@@ -101,6 +103,12 @@ def _handler_provision(command, resources, priority_evaluator, use_kubeconfig, s
             kubeconfig_namespace = list_kube_config_contexts()[1].get('context').get('namespace')
         except Exception as e:
             raise RuntimeError(e)
+    elif in_cluster:
+        try:
+            load_incluster_config()
+            kubeconfig_namespace = list_kube_config_contexts()[1].get('context').get('namespace')
```

Comment:
This repeats twice and not quite pretty code block at all (with those elifs), should be probably refactored somehow in future.